keyboards/keycapsss/kimiko/rev1/info.json

```
@@ -2,106 +2,128 @@
     "keyboard_name": "Kimiko",
     "manufacturer": "Keycapsss",
     "url": "https://keycapsss.com",
-    "maintainer": "@ben_roe Keycapsss",
+    "maintainer": "BenRoe",
     "usb": {
         "vid": "0x7983",
         "pid": "0x4B69",
-        "device_version": "0.0.1"
+        "device_version": "1.0.0"
     },
+    "diode_direction": "COL2ROW",
     "matrix_pins": {
         "cols": ["F6", "F7", "B1", "B3", "B2", "B6"],
         "rows": ["C6", "D7", "E6", "B4", "B5"]
     },
-    "diode_direction": "COL2ROW",
+    "features": {
+        "mousekey": true,
+        "oled": true,
+        "extrakey": true,
+        "encoder": true,
+        "nkro": true,
+        "lto": true
```

lto is enabled in `build`
```suggestion
        "nkro": true
```